src/main/java/org/alfresco/rest/api/search/impl/ResultMapper.java

```diff
@@ -635,4 +642,31 @@ protected SolrJSONResultSet findSolrResultSet(ResultSet results)
 
         return null;
     }
+    public CollectionWithPagingInfo<TupleList> toCollectionWithPagingInfo(JSONArray docs, SearchSQLQuery searchQuery) throws JSONException
```

Comment:
I'm not so sure if this method will be needed at all if you'd implement your REST API using the EntityResource annotation?